go/pkg/sysdb/coordinator/coordinator.go

```diff
@@ -137,6 +137,10 @@ func (s *Coordinator) GetSegments(ctx context.Context, segmentID types.UniqueID,
 	return s.catalog.GetSegments(ctx, segmentID, segmentType, scope, collectionID)
 }
 
+// DeleteSegment is a no-op.
```

Comment:
Do we want to preserve the ability to delete segments and just not error on not-exists? This seems like it could be useful in the future.